spec/models/pglogical_subscription_spec.rb

```diff
@@ -358,4 +358,22 @@ def with_an_invalid_remote_schema
       sub.delete
     end
   end
+  
+  describe "#validate" do
+    it "validates existing subscriptions with new parameters" do
+      allow(pglogical).to receive(:enabled?).and_return(true)
+      allow(pglogical).to receive(:subscriptions).and_return([subscriptions.first], [])
```

Comment:
Is `subscriptions` being called multiple times in this test? If not, you only need to set one response.